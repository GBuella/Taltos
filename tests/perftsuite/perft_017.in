setboard 2r1k2r/8/8/8/8/8/8/R3K2R w KQk - 0 1
echo 2r1k2r/8/8/8/8/8/8/R3K2R w KQk - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
