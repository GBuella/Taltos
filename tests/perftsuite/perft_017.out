2r1k2r/8/8/8/8/8/8/R3K2R w KQk - 0 1
25
560
13592
317324
7710115
185959088
